**Заявляется, что партия изготавливается со средним арифметическим 2,5 см. Проверить данную гипотезу, если известно, что размеры изделий подчинены нормальному закону распределения. Объем выборки 10, уровень статистической значимости 5%.**  

                    2.51, 2.35, 2.74, 2.56, 2.40, 2.36, 2.65, 2.7, 2.67, 2.34

In [1]:
import numpy as np
import scipy.stats as stats

Составим гипотезы:  
$H_0 : М = 2,5$    
$H_1 : М \neq 2.5$

In [2]:
weights = np.array([2.51, 2.35, 2.74, 2.56, 2.40, 2.36, 2.65, 2.7, 2.67, 2.34])
n = 10
mean = weights.mean()
std = weights.std(ddof=1)
mean, std

(2.5279999999999996, 0.1572542173961923)

Поскольку мы проверяем гипотезу относительно математического ожидания нормально распределённой случайной величины с неизвестной дисперсией, будем использовать статистику:  
$t_n = \dfrac{\overline{X} - a}{\sigma_X / \sqrt{n}}$

In [4]:
t_n = (mean-2.5)/(std/np.sqrt(n))
print(f't наблюдаемое равно t_n={t_n: .4f}')

t наблюдаемое равно t_n= 0.5631


Доверительная вероятность равна $0.05$, число степеней свободы равно $n - 1 = 9$

По таблице Стьюдента находим:  
$t_1 =  0.064$  
$t_2 = -0.064$  

Таким образом, $\Omega_\alpha = (-\infty, -0.064) \cup (0.064, \infty).$

Ответ.  
Так как $t_n = 0.5631$ и $ > t_1 (0.064)$ гипотезу $H_0$ отвергаем.  
Заявление, что партия изготавливается со средним арифметическим 2,5 см - _НЕВЕРНО_